cpp/src/parquet/column_reader.cc

```diff
@@ -1218,20 +1218,21 @@ namespace {
 constexpr int64_t kMinLevelBatchSize = 1024;
 
 template <typename DType>
-class TypedRecordReader : public ColumnReaderImplBase<DType>,
+class TypedRecordReader : public TypedColumnReaderImpl<DType>,
```

Comment:
why the change in the base class?

Reply:
To have access to the ColumnReader's Skip method.